os/board/esp32_devkit/src/Makefile

```diff
@@ -89,6 +93,7 @@ else
   CFLAGS += -I$(ARCH_SRCDIR)/chip/rom
   CFLAGS += -I$(BOARD_DIR)/include
   CFLAGS += -I$(TOPDIR)/board/common
+  CFLAGS += -I$(TOPDIR)/../external/esp_idf_port/include
```

Comment:
Why do you need include of external? In architecture point of view, lower layer should not call higher layer. In TizenRT, external positions at top of os.

Reply:
fixed, thanks for suggestion